dsymbol/src/dsymbol/ufcs.d

```diff
@@ -149,13 +305,17 @@ bool isCallableWithArg(const(DSymbol)* incomingSymbol, const(DSymbol)* beforeDot
     if (incomingSymbol.kind == CompletionKind.functionName && !incomingSymbol
         .functionParameters.empty)
     {
-        return beforeDotType is incomingSymbol.functionParameters.front.type
+        if (beforeDotType is incomingSymbol.functionParameters.front.type
+            || incomingSymbol.functionParameters.front.type.kind is CompletionKind.typeTmpParam // non constrained template
             || willImplicitBeUpcasted(beforeDotType.name, incomingSymbol
-                    .functionParameters.front.type.name)
-            || matchAliasThis(beforeDotType, incomingSymbol, recursionDepth);
+                .functionParameters.front.type.name)
+            || matchAliasThis(beforeDotType, incomingSymbol, recursionDepth))
+        {
+            incomingSymbol.kind = CompletionKind.ufcsName;
```

Comment:
don't modify symbols in a function that's called `isXXX`! Also make the parameter const again

Reply:
alright